### PR TITLE
[MNT] - Cleans & checks for new version release. 

### PR DESCRIPTION
**Pull request by TomDonoghue**
Updates from a check through of the code and docs to prepare for a 2.1 release.

### CODE REVIEW

File: neurodsp/spectral/checks.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def check_spg_settings(fs, window, nperseg, noverlap):
         Sampling rate, in Hz.
     window : str or tuple or array_like
         Desired window to use. See scipy.signal.get_window for a list of available windows.
-        If array_like, the array will be used as the window and its length must be nperseg.
+        If array_like, the array will be used as the window and it's length must be nperseg.
```
Comment (ryanhammonds) [inline neurodsp/spectral/checks.py:15]: This should have stayed 'its'.

File: neurodsp/timefrequency/hilbert.py [hunk 2]
```diff
@@ -69,10 +69,10 @@ def phase_by_time(sig, fs, f_range=None, hilbert_increase_n=False,
     f_range : tuple of float or None, optional default: None
         Filter range, in Hz, as (low, high). If None, no filtering is applied.
     hilbert_increase_n : bool, optional, default: False
-        If True, zero pad the signal to length the next power of 2 when doing the Hilbert transform.
+        If True, zero pad the signal to length the next power of 2 for the Hilbert transform.
```
Comment (ryanhammonds) [inline neurodsp/timefrequency/hilbert.py:72]: This reads a bit strange. Maybe "If True, zero pad the signal's length to the next power..."

File: neurodsp/timefrequency/hilbert.py [hunk 3]
```diff
@@ -69,10 +69,10 @@ def phase_by_time(sig, fs, f_range=None, hilbert_increase_n=False,
...
         This is because ``scipy.signal.hilbert`` can be very slow for some lengths of x.
     remove_edges : bool, optional, default: True
-        If True, replace samples that are within half of the filters length to the edge with np.nan.
+        If True, replace samples that are within half of the filters length to the edge with nan.
```
Comment (ryanhammonds) [inline neurodsp/timefrequency/hilbert.py:75]: filter's

File: neurodsp/timefrequency/hilbert.py [hunk 4]
```diff
@@ -118,10 +119,10 @@ def amp_by_time(sig, fs, f_range=None, hilbert_increase_n=False,
     f_range : tuple of float or None, optional default: None
         Filter range, in Hz, as (low, high). If None, no filtering is applied.
     hilbert_increase_n : bool, optional, default: False
-        If True, zero pad the signal to length the next power of 2 when doing the Hilbert transform.
+        If True, zero pad the signal to length the next power of 2 for the Hilbert transform.
         This is because ``scipy.signal.hilbert`` can be very slow for some lengths of sig.
     remove_edges : bool, optional, default: True
-        If True, replace samples that are within half of the filters length to the edge with np.nan.
+        If True, replace samples that are within half of the filters length to the edge with nan.
```
Comment (ryanhammonds) [inline neurodsp/timefrequency/hilbert.py:125]: filter's

File: neurodsp/timefrequency/hilbert.py [hunk 5]
```diff
@@ -118,10 +119,10 @@ def amp_by_time(sig, fs, f_range=None, hilbert_increase_n=False,
     f_range : tuple of float or None, optional default: None
         Filter range, in Hz, as (low, high). If None, no filtering is applied.
     hilbert_increase_n : bool, optional, default: False
-        If True, zero pad the signal to length the next power of 2 when doing the Hilbert transform.
+        If True, zero pad the signal to length the next power of 2 for the Hilbert transform.
```
Comment (ryanhammonds) [inline neurodsp/timefrequency/hilbert.py:122]: This reads a bit strange. Maybe "If True, zero pad the signal's length to the next power..."

File: neurodsp/timefrequency/hilbert.py [hunk 6]
```diff
@@ -167,10 +168,10 @@ def freq_by_time(sig, fs, f_range=None, hilbert_increase_n=False,
...
```
Comment (ryanhammonds) [inline neurodsp/timefrequency/hilbert.py:171]: This reads a bit strange. Maybe "If True, zero pad the signal's length to the next power..."

File: neurodsp/timefrequency/hilbert.py [hunk 7]
```diff
@@ -167,10 +168,10 @@ def freq_by_time(sig, fs, f_range=None, hilbert_increase_n=False,
...
         This is because ``scipy.signal.hilbert`` can be very slow for some lengths of sig.
     remove_edges : bool, optional, default: True
-        If True, replace samples that are within half of the filters length to the edge with np.nan.
+        If True, replace samples that are within half of the filters length to the edge with nan.
```
Comment (ryanhammonds) [inline neurodsp/timefrequency/hilbert.py:174]: filter's

File: tutorials/rhythm/plot_LaggedCoherence.py [hunk 8]
```diff
@@ -42,8 +40,11 @@
 set_random_seed(0)
 
 ###################################################################################################
-# Simulate a Signal with a Bursty Oscillation
-# -------------------------------------------
+# Simulate an Example Signal
+# --------------------------
+#
+# First, we'll simulate an example signal that starts with a burst of alpha activity, followed
+# by a periodic of only aperiodic activity.
```
Comment (ryanhammonds) [inline tutorials/rhythm/plot_LaggedCoherence.py:47]: perodic -> period

File: tutorials/rhythm/plot_LaggedCoherence.py [hunk 9]
```diff
@@ -95,8 +95,9 @@
 # Compute lagged coherence across the frequency spectrum
 # ------------------------------------------------------
 #
-# Notice that lagged coherence peaks around 10Hz (the frequency of our
-# oscillation), but it is not very specific to that frequency.
+# Notice that lagged coherence peaks around 10Hz (the frequency of our oscillation).
+#
+# However, the peak it is not very specific to that frequency.
```
Comment (ryanhammonds) [inline tutorials/rhythm/plot_LaggedCoherence.py:100]: it is -> is

File: neurodsp/spectral/checks.py (hunk 1 above)
Comment (TomDonoghue) [reply neurodsp/spectral/checks.py:15]: lol, good job Tom. Words are hard.

**Comment by ryanhammonds** [comment]
@TomDonoghue I pushed a commit to fix small misc typos I found. If this looks good to you, feel free to merge and I'll push to pypi and rebuild the site.

**Pull request merged**